Gemfile

```diff
@@ -7,6 +7,11 @@ eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 gem "activerecord-deprecated_finders", "~>1.0.4",  :require => "active_record/deprecated_finders"
 gem "rails",                           "~>4.2.3"
 
+# Local gems
+path "gems/" do
+  gem "manageiq_foreman", :require => false
```

Comment:
:+1: didn't occur to me that you could setup a gems path outside the `gem "manageiq...` line